File: car-lease-service/src/main/kotlin/org/jesperancinha/car/lease/model/Customer.java
package org.jesperancinha.car.lease.model;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Entity
@Table
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Customer {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    @Column
    private String name;

    @Column
    private String street;

    @Column
    private Long houseNumber;

    @Column
    private String zipCode;

    @Column
    private String place;

    @Column
    private String email;

    @Column
    private String phoneNumber;

}

File: car-lease-service/src/main/kotlin/org/jesperancinha/car/lease/model/Customer.kt
package org.jesperancinha.car.lease.model

import jakarta.persistence.*

@Entity
@Table
class Customer(
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    val id: Long? = null,

    @Column
    val name: String? = null,

    @Column
    val street: String? = null,

    @Column
    val houseNumber: Long? = null,

    @Column
    val zipCode: String? = null,

    @Column
    val place: String? = null,

    @Column
    val email: String? = null,

    @Column
    val phoneNumber: String? = null
)